### DISCUSSION TITLE
Discord invitation invalid

### DISCUSSION THREAD

**Discussion by Rukkya** [nik132-eng/chrome-extensions#16] [General]
hey, I'm trying to join the discord channel to contribute to the project as a Kwoc participant but the link is invalid could you plz provide a working link to join?

**Comment by nik132-eng** [comment]
Hey I'm checking it and back to you ASAP.

**Comment by nik132-eng** [comment]
Hey @rukkya thank you for mail I updated URL. You can join here

https://discord.com/invite/xPM9Syrk
